cmd/lakefs/cmd/import.go

```diff
@@ -145,4 +166,5 @@ func init() {
 	importCmd.Flags().Bool(DryRunFlagName, false, "Only read inventory and print stats, without making any changes")
 	importCmd.Flags().StringP(ManifestURLFlagName, "m", "", fmt.Sprintf("S3 uri to the manifest.json to use for the import. Format: %s", ManifestURLFormat))
 	_ = importCmd.MarkFlagRequired(ManifestURLFlagName)
+	importCmd.Flags().Bool(WithMergeFlagName, false, "With merge after import completes")
```

Comment:
Helptext: "Merge imported data to the repository's main branch"

Flag suggestion: instead of with-merge >> "merge-to-main" -- but not sure about this one. WDYT?

Reply:
went with the default repository branch - not sure it will be main. also, GitHub remove main so I guess we will be next ;)